catroid/src/main/java/org/catrobat/catroid/content/bricks/DroneMoveRightBrick.java

```diff
@@ -61,12 +61,13 @@ private void initializeBrickFields(Formula durationInSeconds, Formula powerInPer
 
 	public void showFormulaEditorToEditFormula(View view) {
 		switch (view.getId()) {
-			case R.id.brick_drone_move_right_edit_text_second:
-				FormulaEditorFragment.showFragment(view, this, BrickField.DRONE_TIME_TO_FLY_IN_SECONDS);
-				break;
 			case R.id.brick_drone_move_right_edit_text_power:
 				FormulaEditorFragment.showFragment(view, this, BrickField.DRONE_POWER_IN_PERCENT);
 				break;
+			case R.id.brick_drone_move_right_edit_text_second:
+			default:
```

Comment:
stuff like this that doesnt have anything todo with the ticket should be a separate PR, or at least a separate commit

Reply:
Hmmm... Don't know how that stuff got in here, didn't change anything there, will remove it from PR